src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -236,37 +236,71 @@ export default class PhotoField extends React.Component {
     this.setState({ windowWidth });
   }
 
+  updateMoveButtonsEnabledStates = (canvasData, cropboxData) => {
+    const { height: canvasHeight, width: canvasWidth, left: canvasLeft, top: canvasTop } = canvasData;
+    const { height: cropboxHeight, width: cropboxWidth, left: cropboxLeft, top: cropboxTop } = cropboxData;
+
+    // origin is at center
+    // coodinates are for canvas's container
+    const canvasBoundaries = {
```

Comment:
I looked around and couldn't find these values documented. They're returned as part of the `cropper.canvas` object when `move()` is called but I didn't want to rely on something undocumented.

Reply:
Also considered memoizing this object since its values only change on zoom but these calculations are cheap.

Reply:
Those both sound very reasonable.